templates/download/raspberry-pi/index.html

```diff
@@ -372,6 +358,18 @@ <h3>
       </p>
       {% include "download/shared/_core_raspberry_pi_download.html" %}
     </div>
+    <div class="col-6">
+      <h3>
+        Ubuntu Core 22 Beta
+      </h3>
+      <p>
+        Try out the latest version of Ubuntu Core today!
+      </p>
+      <p>
+        <a href="" class="p-button--positive">Download 64-bit</a>
+        <a href="" class="p-button">Download 32-bit</a>
+      </p>
+    </div>    
   </div>
```

Comment:
Looks like this is not available yet. Can we remove for now and leave a note on the copy doc.